src/js/component.js

```diff
@@ -42,13 +51,13 @@ vjs.Component = vjs.CoreObject.extend({
     this.player_ = player;
 
     // Make a copy of prototype.options_ to protect against overriding global defaults
-    this.options_ = vjs.obj.copy(this.options_);
+    this.options_ = vjslib.obj.copy(this.options_);
```

Comment:
Yeah, we don't need both now. utils was meant to house the exported utility functions. Now they're all public.